docs/themes/book/layouts/partials/docs/html-head.html

```diff
@@ -1,10 +1,11 @@
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
-<meta name="description" content="{{ partial "docs/title" . }}">
+<meta name="description" content="{{ default .Summary .Description }}">
+
 {{- template "_internal/opengraph.html" . -}}
 
 <title>{{ partial "docs/title" . }} | {{ .Site.Title -}}</title>
-<link rel="icon" href="{{ "favicon.ico" | relURL }}" type="image/x-icon">
+<link rel="icon" href="{{ "favicon.png" | relURL }}" type="image/x-icon">
```

Comment:
What's wrong with .ico?

Reply:
Nothing wrong. It was a (custom change)[https://github.com/avito-tech/avito-android/pull/304/files] in the theme.
Any customization increases the price of updates. I've decided to fit the default one.